doc/userdoc/contribute/templates_styleguides/styleguide.rst

```diff
@@ -0,0 +1,608 @@
+
+
+To reference figures or arbitrary places in a file, you must include a custom
+label in the reference for it to work::
+
+    :ref:`custom label <my-arbitrary-place-label>`
+
+
+
+Special links
+~~~~~~~~~~~~~
+
+.. attention::
+
+  The items in this section are still in development and have not been incorporated into nest:master!
+
+Links to functions
+^^^^^^^^^^^^^^^^^^
+
+To link PyNEST API functions used in-text to the API reference page use the following syntax::
+
+   :py:func:`.Create`
+
+Rendered as
+
+   :py:func:`.Create`
+
+Classes, methods etc. can also be linked: ``:py:class:`` ``:py:meth:``.
+If you want to explictly show a complete function call, like ``nest.Create("iaf_psc_apha")``, the link cannot be used.
+
+.. note::
+
+   Functions within classes NodeCollection and SynapseCollection require different syntax as follows
+
+   ``:py:func:`~nest.lib.hl_api_types.SynapseCollection.funcname```
+
+   ``:py:func:`~nest.lib.hl_api_types.NodeCollection.funcname```
+
+
+Links to glossary
+^^^^^^^^^^^^^^^^^
+
+To link terms to the glossary page use the HoverXTooltip role :hxt_ref: from Mahdi Enan (INM-6)::
+
+  :hxt_ref:`E_L`
+
+Links to certain external projects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With the Sphinx extension 'intersphinx', projects that also use Sphinx/ReadtheDocs can be referenced the same way as
+your local project. You can use the reference label role (``:ref:``), document role (``:doc:``), and Python role
+(``:py:func:``, ``:py:class:``). You only need to add the intersphinx unique identifer to the reference, which
+looks like this ``:doc:`custom label <unique-identifier:filename>```. See section in userdoc/conf.py "intersphinx_mapping" to
+see which projects are currently included along with their unique identifier.
+
+Examples of syntax::
+
+  :doc:`tutorial for nestml <nestml:tutorials>`
+  :py:func:`pyNN.utility.get_simulator`
+
+
+.. note::
+
+   Depending on how a project is documented, you may only be able to use the ``:doc:`` role or the ``:ref:`` role.
+   To find out, you need to look into the objects.inv file, which can be obtained with the following code
+
+   ``python -msphinx.ext.intersphinx https://docs.project.org/objects.inv``
+
+   Objects in objects.inv are categorized into different sections.
+   The std:label refers to objects that use the ``:ref:`` role. And std:doc refers to objects that use the ``:doc:`` role.
+
+Bibliography style
+------------------
+
+The reStructuredText bibliography style is used throughout documentation so links
+are autogenerated and a consistent format is used.
+
+For in-text citations, we use the reStructuredText numeric style ``[1]_``.
+
+For example:
+
+    The following example is based on Smith [1]_.
+    Sanders et al. [2]_ contains a technically detailed example.
+
+Please ensure your reference follows the following guidelines:
+
+*  Do not add formatting markup such as italics, bold, or underline.
```

Comment:
Not rendered as list.

Reply:
I'm not sure why this is - the bullets render correctly for me.